src/data-fetcher-loop/data-fetcher-loop.ts

```diff
@@ -83,7 +83,7 @@ export const runDataFetcher = async () => {
 
     const urlCount = urls.length;
     const staggerTimeMs = signedDataFetchIntervalMs / urlCount;
-    logger.info('Fetching signed data.', { urlCount, staggerTimeMs });
+    logger.info('Fetching signed data.', { urlCount, staggerTimeMs, currentTime: new Date().toISOString() });
```

Comment:
This message is logged as a basically the first thing, so I've logged the current time here instead of logging another message.